index.md

```diff
@@ -105,26 +105,26 @@ Taxonomic classifiers perform best when they are trained based on your specific
 ```
 
 ```{card}
-:header: **Greengenes2 2022.10 full length sequences**
+:header: **Greengenes2 2024.09 full length sequences**
 
-**Download**: [Greengenes2 2022.10 full length sequences](https://data.qiime2.org/classifiers/sklearn-1.4.2/greengenes2/2022.10.backbone.full-length.nb.sklearn-1.4.2.qza)\
-**UUID**: 1df1eab1-2ee4-4d92-917e-ba1f6c937269\
-**SHA256**: 4f7ab05b57f85a76b12049c057c363759112fe17b7337f69f1ab2db0ec668024\
+**Download**: [Greengenes2 2024.09 full length sequences](https://data.qiime2.org/classifiers/sklearn-1.4.2/greengenes2/2024.09.backbone.full-length.nb.sklearn-1.4.2.qza)\
+**UUID**: 49ccfb0a-155d-404b-80b4-818d2aeb53b2\
+**SHA256**: dc278214bf5e493d845b55ddf7e572d9\
```

Comment:
So I don't know anything about SHAs.
These values seems alot shorter than the orginal SHA in the diff and 
My value when I run: 
shasum -a 256 ~/Downloads/2024.09.backbone.full-length.nb.sklearn-1.4.2.qza

 
5c5900c10ad601b12cf0d2280d52cc0f7852627e89070595cc3899a6d690ee27  
2024.09.backbone.full-length.nb.sklearn-1.4.2.qza